README.md

```diff
@@ -438,19 +443,21 @@ pass this object to the client when starting it, as shown below.
 
 ```C++
     hazelcast::client::client_config config;
-    hazelcast::client::hazelcast_client hz(config); // Connects to the cluster
+    config.set_cluster_name("my-cluster"); // the server is configured to use the `my_cluster` as the cluster name hence we need to match it to be able to connect to the server.
+    config.get_network_config().add_address(address("192.168.1.10", 5701));    
+    hazelcast::client::hazelcast_client hz(std::move(config)); // Connects to the cluster member at ip address `192.168.1.10` and port 5701
 ```
 
 If you run the Hazelcast IMDG members in a different server than the client, you most probably have configured the members' ports and cluster
 names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
 
-### 1.4.2.1 Group Settings
+### 1.4.2.1 Cluster Name
 
-You need to provide the group name of the cluster, if it is defined on the server side, to which you want the client to connect.
+You only need to provide the name of the cluster if it is explcitely configured at the server side (otherwise the default value of `dev` is used).
```

Comment:
```suggestion
You need to provide the name of the cluster only if it is explicitly configured on the server-side (otherwise the default value of `dev` is used).
```